maia-core/src/interval.rs

```diff
@@ -14,7 +14,7 @@ const MAX_DIGITS: usize = 20;
 const BASE: usize = 2;
 
 /// Binary representation of a price interval.
-#[derive(Clone, Debug, PartialEq)]
```

Comment:
🤔a new clippy lint from new rustc version?

Reply:
This is only for `beta` target. I'm not sure why we even build it for `beta` 😅